### DISCUSSION TITLE
mode self atau mode admin / cara pasang prefix

### DISCUSSION THREAD

**Discussion by dammingyu** [ImYanXiao/Elaina-MultiDevice#61] [General]
bre ini cara pasang prefixnya gimana?
dan juga ini cara mode self atau mode admin juga gimana?

**Comment by ImYanXiao** [comment]
Wah Bingung sih :v

**Comment by ImYanXiao** [comment]
> bre ini cara pasang prefixnya gimana? dan juga ini cara mode self atau mode admin juga gimana?

Okey, Kalo Prefix Ada Di Main.js bagian global.prefix
Klo self bisa aja ditambahin dibagian opts
